app.rb

```diff
@@ -122,4 +124,26 @@ def list_genres
   def add_music_album
     @music.add_music_album
   end
+
+  def save_authors
+    author_add = []
+    @author_list.each do |author|
+      author_add << {
+        id: author.id,
+        first_name: author.first_name,
+        last_name: author.last_name
+      }
+    end
```

Comment:
- [ ] To keep your application more straightforward and make it work I would advise you to give the author 1 name instead of a first and last name, if you really want to implement this then you should make sure to ask the user for a first and last name and adjust either the book class or combine the first and last name before adding it to the book class.